syntax = "proto3";

package rules;

message CarryRequest {
    string symbol = 1;
    int32 smooth_days = 2;
    bool use_attenuation = 3;
    float scaling_factor = 4;
    string scaling_type = 5;

}

message CarryResponse {
    bytes series = 1;
}

service Carry{
    rpc get_carry (CarryRequest) returns (CarryResponse);
}